### DISCUSSION TITLE
Welcome to OpenGloves Discussions!

### DISCUSSION THREAD

**Discussion by danwillm** [LucidVR/opengloves-driver#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to troubleshoot, make suggestions for features and to ask more general questions that may not as a GitHub Issue.

  * Ask questions you’re wondering about.
  * Share ideas.
  * Any discussion you want that you feel may not be appropriate as a GitHub Issue
  
  To get started, drop us a message below with the question/suggestion and we'll be happy to help out!

We actively welcome and encourage all discussions that are in the breadth of this project.

With that in mind, please make the upmost effort to be excellent to eachother and keep discussions to appropriate channels.

**Comment by timbomur** [comment]
can you direct me to the troubleshooting thread?  I have built the gloves but the fingers aren't moving in Alyx.

**Reply by danwillm** [reply]
hey - have you set the bindings for the gloves in steamvr? You'll need to do that first before moving onto trying in games.

If so, please do open a thread in the Help & Troubleshooting category, or open an issue.

**Comment by Flurior** [comment]
I have a question, is there an option to use your gloves within a motion capturing software?

**Reply by danwillm** [reply]
If the motion capturing software is compatible with SteamVR it should work, but feel free to open an issue on the hardware repo or this one if you think this is something that you would like to be supported.

**Comment by badaaim** [comment]
Hiya Lucas, Hiya Dan,

Really amazed at your work and how well everything works, especially with regards to the finger tracking and your price point so far. I have a couple of questions
1. How did you figure out the configuration and parameters for the knucklesdriver for knuckles controller emulation?
2. We had a version of our custom driver with skeleton/finger tracking which worked for Moondust VR and some games but not for Half-life Alyx. Whereas it seems to be the opposite for you where your driver works natively for Alyx but not for games that use Legacy Binding such as VR Chat. Would you have any insight on that?

**Reply by danwillm** [reply]
Thanks!

1. We fetched them from the controller's properties you can get from the device in openvr. I'm sure it's not really needed to set that many properties - I'm sure games don't really care about most of them but we did it anyway.
2. I'm not too sure why that's not working for you. Our driver works well with finger tracking for all those games you mentioned, even ones using legacy bindings like VRChat.

What sort of thing are you developing? It would be nice to have a talk if you're doing the same sort of thing.

**Reply by badaaim** [reply]
Hiya, apologies for late reply. We are creating eteeControllers https://eteexr.com/
1. I see thank you for the information
2. I managed to resolve it eventually. I don't know how this is going to work for end-users eventually but for Superhot VR at least, inside their folder I needed to create a new binding_etee.json file and similarly reference that file inside their actions.json file under the "default bindings" section. 

Really amazing work by the way, I have been following your development and recently saw your steam app for the driver installation. 

On a separate note, I am currently working on the custom reference poses for our controller.

**Reply by danwillm** [reply]
ah nice - I've heard about etee, very cool!

Yes, we just published the driver onto steam - we've found it a much better option for letting users install the driver (it's just one click and we can ship a UI with it which helps a lot)

For us, we've kinda circumvented those controller issues with games not wanting to use custom controllers by pretending to be knuckle controllers - I _think_ you might find that your controllers don't behave properly in Boneworks, VRChat, etc. with finger tracking... but it would be interesting to see if they do (we might have done something wrong).

I'd love to talk more about this, you're welcome to reach me on discord: `danwillm#8254`

**Reply by lucas-vrtech** [reply]
Adding to this, feel free to join the discord server to meet with more people involved with the project and spin-offs based on it  😊https://discord.gg/PjwrhhHN

**Reply by badaaim** [reply]
Yeah I looked at the console, I remember previously you had to specify where to find the json file to edit and what to edit to configure the driver haha, definitely the current UI is so much more user friendly 

Really rooting for you guys. 

We are actually at a limbo regarding the finger tracking. So we haven't really reached the stage for Boneworks and VR chat, currently it behaves more like a Vive controller. 

Cool, will join discord and provide more info there.
